docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+
+For all of the config sections specific to each test case, users need to define
+the interpolation scheme among ``bilinear``, ``neareststod`` and ``conserve``
+methods (however, note that the ocean basal test case should always use the
+``neareststod`` method because the source files have a single valued data per
+basin.)
+
+Below is the default config options:
+
+.. code-block:: cfg
+
+    # config options for ismip6 antarctic ice sheet data set
+    [ismip6_ais]
+
+    # Base path to the input ismip6 ocean and smb forcing files. User has to supply.
+    base_path_ismip6 = NotAvailable
+
+    # Base path to the the MALI mesh. User has to supply.
+    base_path_mali = NotAvailable
+
+    # Forcing end year of the ISMIP6 data. User has to supply.
+    # Available end years are 2100 and 2300.
+    period_endyear = NotAvailable
+
+    # Base path to which output forcing files are saved.
+    output_base_path = NotAvailable
+
+    # Name of climate model name used to generate ISMIP6 forcing data. User has to supply.
+    # Available model names are the following: CCSM4, CESM2-WACCM, CSIRO-Mk3-6-0, HadGEM2-ES, NorESM1-M, UKESM1-0-LL
+    model = NotAvailable
+
+    # Scenarios used by climate model. User has to supply.
+    # Available scenarios are the following: RCP26, RCP26-repeat, RCP85, SSP126, SSP585
+    scenario = NotAvailable
+
+    # name of the mali mesh. User has to supply. Note: It is used to name mapping files.
+    # (e,g. 'map_ismip6_8km_to_{mali_mesh_name}_{method_remap}.nc').
+    mali_mesh_name = NotAvailable
+
+    # MALI mesh file to be used to build mapping file (netCDF format). User has to supply.
+    mali_mesh_file = NotAvailable
+
+    # config options for ismip6 antarctic ice sheet SMB forcing data test cases.
+    [ismip6_ais_atmosphere]
+
+    # Remapping method used in building a mapping file. Options include: bilinear, neareststod, conserve
+    method_remap = NotAvailable
+
+    # Set True to process RACMO modern climatology.
+    process_smb_racmo = NotAvailable
+
+    # Full path to the RACMO regional climate model SMB data for the modern climatology.
+    base_path_racmo = NotAvailable
+
+    # config options for ismip6 ocean thermal forcing data test cases.
+    [ismip6_ais_ocean_thermal]
+
+    # Remapping method used in building a mapping file. Options include: bilinear, neareststod, conserve
+    method_remap = NotAvailable
+
+    # Set to True if the want to process observational thermal forcing data.
+    # Set to False if want to process model thermal forcing data.
+    # Note: when set True, the ['ismip6_ais'] config options 'period_endyear', 'model' and 'scenario' will be ignored.
+    process_obs_data = NotAvailable
+
+    # config options for ismip6 ocean basal test case
+    [ismip6_ais_ocean_basal]
+
+    # Remapping method used in building a mapping file. Ocean basal testcase will always want to
+    # use neareststod method.
+    method_remap = NotAvailable
```

Comment:
```suggestion
    method_remap = neareststod
```
@hollyhan, could you check that this whole namelist is accurate by just copying and pasting it again?